import java.util.*;
public class evenNumberInArray {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
System.out.print("Enter the number of element in a array:");
int n= sc.nextInt();
int[]arr=new int[n];
for(int i=0;i<n;i++){
System.out.print("Enter the"+(i+1)+"Element:");
arr[i]=sc.nextInt();
}
System.out.println("even element are:");
for(int i=0;i<n;i++)
{
    if(arr[i]%2==0)
    {
        System.out.println(arr[i]+"");
}
    } 
    }
}
